packages/angularjs-extensions/README.md

```diff
@@ -6,13 +6,14 @@ This module can be used as template to extend existing RainCatcher user interfac
 
 ## Extending workorders
 
-Extending workorders data is possible by providing implementation for 2 directives:
+RainCatcher by default provides minimal metadata for workorders.
+Depending on business developers can add additional data like gps location, address, description. This data can be used for preview and informational purposes or even for route optimalization and scheduling workorders.
```

Comment:
`Depending on business requirements, developers can add additional metadata like gps, location, address and description.`